### PR TITLE
Petra Söderström - Project-portfolio, Week 4

### PR DESCRIPTION
**Pull request by Petrasoderstrom1612**
https://velvety-choux-4c3752.netlify.app/

### CODE REVIEW

**Comment by Petrasoderstrom1612** [comment]
https://velvety-choux-4c3752.netlify.app/

File: index.html [hunk 1]
```diff
@@ -0,0 +1,402 @@
...
+    <link rel="stylesheet" href="style.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
+    <!-- <link rel="icon" type="image/x-icon" href="./assets/favicon.png" /> -->
+    <link
+      rel="apple-touch-icon"
+      sizes="180x180"
+      href="images/apple-touch-icon.png"
```
Comment (saralieb) [inline index.html:16]: nice with the personalized icon

File: index.html [hunk 2]
```diff
@@ -0,0 +1,402 @@
...
+      rel="icon"
+      type="image/png"
+      sizes="16x16"
+      href="images/favicon-16x16.png"
+    />
+    <title>Petra Söderström</title>
+  </head>
+  <body>
+    <header class="main-header">
+      <div class="logos">
```
Comment (saralieb) [inline index.html]: these logos are not focusable on keyboard add ```tabindex="0"```

File: index.html [hunk 3]
```diff
@@ -0,0 +1,402 @@
...
+        </div>
+      </div>
+    </div>
+
+    <!------------------TECH ---------------------- -->
+    <section class="outer-container white">
+      <div class="content-container">
+        <h2 class="headline" tabindex="0">TECH</h2>
+        <p tabindex="0">
+          <span class="important-languages">HTML</span>,
```
Comment (saralieb) [inline index.html]: cool that you focus on the key languages

File: index.html [hunk 4]
```diff
@@ -0,0 +1,402 @@
...
+          <h3 class="smaller-headline" tabindex="0">
+            WEATHER FORECAST BUILT IN JAVASCRIPT
+          </h3>
+          <p class="picture-paragraph" tabindex="0">
+            Responsive website using HTML, CSS, JS and APIs in order to fetch
+            the latest updates about the weather forecast for the week in
+            various cities
+          </p>
+        </div>
+        <div class="language-tags">
```
Comment (saralieb) [inline index.html]: these  language tags are not keyboard-focusable

File: index.html [hunk 5]
```diff
@@ -0,0 +1,402 @@
...
+          responsive design,
+          <span class="important-languages">JavaScript</span> - ES6,
+          <!-- JSX, React, React Hooks, Redux, Node.js, Mongo DB,-->
+          Web Accessibly, API:s, mob-programming, pair-programming, Github,
+          Figma, Miro.
+        </p>
+      </div>
+    </section>
+
+    <!------------------FEATURED PROJECTS ---------------------- -->
```
Comment (saralieb) [inline index.html]: It would be nice to have a link to the features projects. The links are only in the Other project section-

File: index.html [hunk 6]
```diff
@@ -0,0 +1,402 @@
...
+          <p tabindex="0"><span class="tags-inline-background">JS</span></p>
+          &nbsp;&nbsp;
+          <p tabindex="0"><span class="tags-inline-background">APIs</span></p>
+        </div>
+      </div>
+      <div class="content-container">
+        <div class="pictures-project fox-flex">
+          <img
+            src="images/shopsite.PNG"
+            alt="A picture of a start site to order a shirt"
```
Comment (saralieb) [inline index.html]: very clear alt description

File: index.html [hunk 7]
```diff
@@ -0,0 +1,402 @@
...
+          <p tabindex="0"><span class="tags-inline-background">HTML</span></p>
+          &nbsp;&nbsp;
+          <p tabindex="0"><span class="tags-inline-background">CSS</span></p>
+          &nbsp;&nbsp;
+          <p tabindex="0"><span class="tags-inline-background">JS</span></p>
+        </div>
+      </div>
+
+      <div class="content-container">
+        <h3 class="headline-other-projects" tabindex="0">GUESS WHO GAME</h3>
```
Comment (saralieb) [inline index.html]: I would suggest to have the most recent projects in the Featured project section and the oldest on in the More Project section. Then the recruiter will see have a better view of your skills.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,402 @@
...
+
+    <!------------------MY THOUGHTS ---------------------- -->
+    <article class="outer-container white">
+      <div class="content-container">
+        <h2 class="headline headline-code" tabindex="0">
+          MY THOUGHTS ABOUT CODE
+        </h2>
+        <div class="pictures-project">
+          <img
+            src="images/newssite.PNG"
```
Comment (saralieb) [inline index.html]: image not seen on desktop

File: index.html [hunk 9]
```diff
@@ -0,0 +1,402 @@
...
+          designing cosmetics, clothes websites, jewelerry shops just into
+          bringing the little feminine creativity touch to every project. I
+          believe diversity is the key to bring powerful ideas together in order
+          to create something big.
+        </p>
+      </div>
+    </article>
+
+    <!------------------MORE THOUGHTS ---------------------- -->
+    <section class="outer-container white">
```
Comment (saralieb) [inline index.html]: nice extra touch with this section

File: index.html [hunk 10]
```diff
@@ -0,0 +1,402 @@
...
+    </section>
+
+    <!------------------SKILLS ---------------------- -->
+    <section class="outer-container">
+      <div class="content-container">
+        <h2 class="headline" tabindex="0">SKILLS</h2>
+        <div class="skills-flex">
+          <div>
+            <h3 class="skills-headline" tabindex="0">
+              &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;CODE
```
Comment (saralieb) [inline index.html]: what does this do? ```&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;``` Is it necessary to have it?

**Review by Scarylie — APPROVED** [review]
I think you have done a really good job with your portfolio. There is just some tips i wrote; such as featuting your most recent project and not the first project thet you have done and also provide a link to those projects. Some links are not keyboard-focusable so that would be good to check it out and also to have a hover effect when hovering a link. I like that you have spent time writing your thoughts and made apersonal touch to the profile and not just focusing on the code. Good job!

**Pull request closed without merging**